misc/python/materialize/cloudtest/k8s/testdrive.py

```diff
@@ -65,9 +65,10 @@ def run(
         seed: Optional[int] = None,
         caller: Optional[Traceback] = None,
         default_timeout: str = "300s",
+        log_level: str = "off",
```

Comment:
Let's keep consistent naming of `log_filter` throughout .

Reply:
✅ Renamed